client/lib/login/index.js

```diff
@@ -256,5 +256,24 @@ export const getPluginTitle = ( pluginName, translate ) => {
 		default: translate( 'Jetpack' ),
 	};
 
-	return pluginNames[ pluginName ] || pluginNames.default;
+	if ( ! pluginName ) {
```

Comment:
Nit: What do you think about renaming `pluginNames` to something like `allowedPluginNames` here since `pluginName` is too close but they convey a different meaning.